python/mxnet/contrib/quantization.py

```diff
@@ -552,9 +552,9 @@ def quantize_model_mkldnn(sym, arg_params, aux_params, data_names=('data',),
         raise ValueError('currently only supports single ctx, while received %s' % str(ctx))
     if ctx.device_type != 'cpu':
         raise ValueError(
-            'quantize_model_mkldnn only support Intel cpu platform with MKL-DNN Backend')
+            'quantize_model_dnnl only support Intel cpu platform with DNNL Backend')
 
-    sym = sym.optimize_for(backend='MKLDNN_QUANTIZE')
+    sym = sym.optimize_for(backend='DNNL_QUANTIZE')
```

Comment:
```suggestion
    sym = sym.optimize_for(backend='ONEDNN_QUANTIZE')
```

Reply:
@anko-intel, I would suggest to aling on `ONEDNN` everywhere.

Reply:
@vpirogov , Originally I also think thought we can move to use oneDNN name everywhere, but from https://oneapi-src.github.io/oneDNN/v2/dev_guide_transition_to_dnnl.html I can read that DNNL name will be still used. For example DNNL_VERBOSE. This way we have little bit strange situation when we describe something as oneDNN and use for it DNNL_something later.  It happens for runtime and for cmake. Are you going to add additional flags and environment name with ONEDNN prefix?

Reply:
This is a good point. We had this in plan for consistency, but did not implement the change yet. I'll plan this for oneDNN v2.5 release.